javalin/src/main/java/io/javalin/http/Context.kt

```diff
@@ -62,14 +62,11 @@ interface Context {
     /** Servlet response */
     fun res(): HttpServletResponse
 
-    /** Gets the handler type of the current handler */
-    fun handlerType(): HandlerType
+    /** Gets all endpoints visited during this request (in order) */
+    fun endpoints(): List<Endpoint>
```

Comment:
I think it should return `List<out Endpoint>`, because:

1. `Endpoint` is open class, so we allow to return various types here
2. From Java point of view - this interface is still mutable, because you can add elements to it. By using a covariant we'll ensure it's readonly on type level.

Reply:
oh, Kotlin is already doing it under the hood :0 so it's already readonly in Java as well niiice, then ignore me